### ISSUE TITLE
rsa.private is a directory that can't be written

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/central#89]
In my fresh install of Central v0.5, there is an empty `/files/dkim/rsa.private` folder.  The docs at https://docs.opendatakit.org/central-install-digital-ocean/#configuring-dkim say that I should run `openssl genrsa -out rsa.private 1024` in `~/central/files/dkim`, but that command fails with `genrsa: Can't open "rsa.private" for writing, Is a directory`.

**Comment by issa-tseng** [comment]
hm. i don't know how? [there isn't one in the repo](https://github.com/opendatakit/central/tree/master/files/dkim) and docker does not write to the host filesystem.

**Comment by yanokwa** [comment]
Is it possible that docker-compose is doing it? https://github.com/opendatakit/central/blob/master/docker-compose.yml#L15

**Comment by issa-tseng** [comment]
hmmmm it's not supposed to, and that was the config i got from the instructions for namshi-smtp. but i'll look into it later.

**Comment by issa-tseng** [comment]
(if the thing doesn't exist on the host docker is supposed to fail or ignore.)

**Comment by issa-tseng** [comment]
okay, it looks like this is Just The Way Docker Works (because legacy reasons) when using this particular configuration format and the `--mount` option that doesn't do things this way is unavailable to us in docker-compose. so we may just have to solve the problem at the documentation level and tell people to remove the directories and rm the image.

there is an alternative path where we can use named volumes that are manually created on the host via `docker` commands, and referenced in the configuration file. but i don't know what it would do if the named volume is absent, and this methodology would be incompatible with existing installations, so anybody with dkim already active will have it break on them (probably silently) unless they do some additional work.

long story short: there are no good answers here.

**Comment by issa-tseng** [comment]
resolved via documentation in https://github.com/opendatakit/docs/pull/1132
